package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/plugin-hooks",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
I know this is a dummy change, but can we bump the patch version instead?

Reply:
ok like 0.1.1?